packages/rocketchat-cloud/server/functions/retrieveRegistrationStatus.js

```diff
@@ -0,0 +1,18 @@
+export function retrieveRegistrationStatus() {
+	const info = {
+		registeredWithWizard: RocketChat.settings.get('Register_Server'),
+		workspaceConnected: (RocketChat.settings.get('Cloud_Workspace_Client_Id')) ? true : false,
+		userAssociated: (RocketChat.settings.get('Cloud_Workspace_Account_Associated')) ? true : false,
+		token: '',
+		email: '',
+	};
+
+	const firstUser = RocketChat.models.Users.getOldest({ emails: 1 });
+	info.email = firstUser && firstUser.emails[0].address;
+
+	if (RocketChat.models.Settings.findOne('Organization_Email')) {
```

Comment:
this query will always return the setting record, even if its value is blank, as the setting will always be there.. should this be testing its value i.e.: `RocketChat.settings.get('Organization_Email')` ?